lib/glimesh_web/live/homepage_live.html.leex

```diff
@@ -5,7 +5,11 @@
       <div class="col-md-12 p-lg-5 mx-auto my-5">
         <h1 class="display-4 font-weight-normal"><%= gettext("Next-Gen Live Streaming") %></h1>
         <p class="lead font-weight-normal"><%= gettext("The first live streaming platform built around truly real time interactivity. Our streams are warp speed, our chat is blazing, and our community is thriving. We're launching soon, but you can get your account setup early and learn more about us below!") %></p>
+        <%= if @current_user do %>
+        <%= link gettext("Edit Your Account"), to: "/users/settings/profile", class: "btn btn-primary btn-lg mt-3" %>
```

Comment:
Only thing I would do here *but its very nit picky* is to indent this line and line 11 since their inside the if else statements. I just feel its better for code readability when it comes to html code even if its elixir generated html.

Reply:
I fixed it. I made the divs go at an angle like the others in the file as well.